### ISSUE TITLE
BrutException

### ISSUE THREAD

**Issue by linuxexp** [iBotPeaches/Apktool#1687]
### Information
1. **Apktool Version (`apktool -version`)** - 2.3.0
2. **Operating System (Mac, Linux, Windows)** - Mac
3. **APK From? (Playstore, ROM, Other)** - Playstore

### Stacktrace/Logcat

```
I: Using Apktool 2.3.0 on base.apk
I: Loading resource table...
I: Decoding AndroidManifest.xml with resources...
I: Loading resource table from file: /Users/raja/Library/apktool/framework/1.apk
I: Regular manifest package...
I: Decoding file-resources...
I: Decoding values */* XMLs...
I: Baksmaling classes.dex...
I: Baksmaling classes2.dex...
I: Copying assets and libs...
I: Copying unknown files...
I: Copying original files...
$ ~/cordovaTest $ java -jar apktool_2.3.0.jar b base/ -o base.breaked.apk
I: Using Apktool 2.3.0
I: Checking whether sources has changed...
I: Smaling smali folder into classes.dex...
I: Checking whether sources has changed...
I: Smaling smali_classes2 folder into classes2.dex...
I: Checking whether resources has changed...
I: Building resources...
W: /Users/raja/cordovaTest/base/res/values-v24/styles.xml:7: error: Error retrieving parent for item: No resource found that matches the given name '@android:style/Animation.OptionsPanel'.
W:
W: /Users/raja/cordovaTest/base/res/values-v24/styles.xml:8: error: Error retrieving parent for item: No resource found that matches the given name '@android:style/Animation.LockScreen'.
W:
Exception in thread "main" brut.androlib.AndrolibException: brut.androlib.AndrolibException: brut.common.BrutException: could not exec (exit code = 1): [/var/folders/88/c8f0s39x66q5qc1fsgqmf62m0000gn/T/brut_util_Jar_1429171880428512863.tmp, p, --forced-package-id, 127, --min-sdk-version, 15, --target-sdk-version, 25, --version-code, 235, --version-name, 1.0.104, --no-version-vectors, -F, /var/folders/88/c8f0s39x66q5qc1fsgqmf62m0000gn/T/APKTOOL5472789814033806451.tmp, -0, arsc, -0, arsc, -I, /Users/raja/Library/apktool/framework/1.apk, -S, /Users/raja/cordovaTest/base/res, -M, /Users/raja/cordovaTest/base/AndroidManifest.xml]
	at brut.androlib.Androlib.buildResourcesFull(Androlib.java:485)
	at brut.androlib.Androlib.buildResources(Androlib.java:419)
	at brut.androlib.Androlib.build(Androlib.java:318)
	at brut.androlib.Androlib.build(Androlib.java:270)
	at brut.apktool.Main.cmdBuild(Main.java:224)
	at brut.apktool.Main.main(Main.java:75)
Caused by: brut.androlib.AndrolibException: brut.common.BrutException: could not exec (exit code = 1): [/var/folders/88/c8f0s39x66q5qc1fsgqmf62m0000gn/T/brut_util_Jar_1429171880428512863.tmp, p, --forced-package-id, 127, --min-sdk-version, 15, --target-sdk-version, 25, --version-code, 235, --version-name, 1.0.104, --no-version-vectors, -F, /var/folders/88/c8f0s39x66q5qc1fsgqmf62m0000gn/T/APKTOOL5472789814033806451.tmp, -0, arsc, -0, arsc, -I, /Users/raja/Library/apktool/framework/1.apk, -S, /Users/raja/cordovaTest/base/res, -M, /Users/raja/cordovaTest/base/AndroidManifest.xml]
	at brut.androlib.res.AndrolibResources.aaptPackage(AndrolibResources.java:454)
	at brut.androlib.Androlib.buildResourcesFull(Androlib.java:471)
	... 5 more
Caused by: brut.common.BrutException: could not exec (exit code = 1): [/var/folders/88/c8f0s39x66q5qc1fsgqmf62m0000gn/T/brut_util_Jar_1429171880428512863.tmp, p, --forced-package-id, 127, --min-sdk-version, 15, --target-sdk-version, 25, --version-code, 235, --version-name, 1.0.104, --no-version-vectors, -F, /var/folders/88/c8f0s39x66q5qc1fsgqmf62m0000gn/T/APKTOOL5472789814033806451.tmp, -0, arsc, -0, arsc, -I, /Users/raja/Library/apktool/framework/1.apk, -S, /Users/raja/cordovaTest/base/res, -M, /Users/raja/cordovaTest/base/AndroidManifest.xml]
	at brut.util.OS.exec(OS.java:95)
	at brut.androlib.res.AndrolibResources.aaptPackage(AndrolibResources.java:448)
	... 6 more
$ ~/cordovaTest $ java -jar apktool_2.3.0.jar b base/ -o base.breaked.apk --force
I: Using Apktool 2.3.0
I: Smaling smali folder into classes.dex...
I: Smaling smali_classes2 folder into classes2.dex...
I: Building resources...
W: /Users/raja/cordovaTest/base/res/values-v24/styles.xml:7: error: Error retrieving parent for item: No resource found that matches the given name '@android:style/Animation.OptionsPanel'.
W:
W: /Users/raja/cordovaTest/base/res/values-v24/styles.xml:8: error: Error retrieving parent for item: No resource found that matches the given name '@android:style/Animation.LockScreen'.
W:
Exception in thread "main" brut.androlib.AndrolibException: brut.androlib.AndrolibException: brut.common.BrutException: could not exec (exit code = 1): [/var/folders/88/c8f0s39x66q5qc1fsgqmf62m0000gn/T/brut_util_Jar_2605467941360292710.tmp, p, --forced-package-id, 127, --min-sdk-version, 15, --target-sdk-version, 25, --version-code, 235, --version-name, 1.0.104, --no-version-vectors, -F, /var/folders/88/c8f0s39x66q5qc1fsgqmf62m0000gn/T/APKTOOL2620364219623722891.tmp, -0, arsc, -0, arsc, -I, /Users/raja/Library/apktool/framework/1.apk, -S, /Users/raja/cordovaTest/base/res, -M, /Users/raja/cordovaTest/base/AndroidManifest.xml]
	at brut.androlib.Androlib.buildResourcesFull(Androlib.java:485)
	at brut.androlib.Androlib.buildResources(Androlib.java:419)
	at brut.androlib.Androlib.build(Androlib.java:318)
	at brut.androlib.Androlib.build(Androlib.java:270)
	at brut.apktool.Main.cmdBuild(Main.java:224)
	at brut.apktool.Main.main(Main.java:75)
Caused by: brut.androlib.AndrolibException: brut.common.BrutException: could not exec (exit code = 1): [/var/folders/88/c8f0s39x66q5qc1fsgqmf62m0000gn/T/brut_util_Jar_2605467941360292710.tmp, p, --forced-package-id, 127, --min-sdk-version, 15, --target-sdk-version, 25, --version-code, 235, --version-name, 1.0.104, --no-version-vectors, -F, /var/folders/88/c8f0s39x66q5qc1fsgqmf62m0000gn/T/APKTOOL2620364219623722891.tmp, -0, arsc, -0, arsc, -I, /Users/raja/Library/apktool/framework/1.apk, -S, /Users/raja/cordovaTest/base/res, -M, /Users/raja/cordovaTest/base/AndroidManifest.xml]
	at brut.androlib.res.AndrolibResources.aaptPackage(AndrolibResources.java:454)
	at brut.androlib.Androlib.buildResourcesFull(Androlib.java:471)
	... 5 more
Caused by: brut.common.BrutException: could not exec (exit code = 1): [/var/folders/88/c8f0s39x66q5qc1fsgqmf62m0000gn/T/brut_util_Jar_2605467941360292710.tmp, p, --forced-package-id, 127, --min-sdk-version, 15, --target-sdk-version, 25, --version-code, 235, --version-name, 1.0.104, --no-version-vectors, -F, /var/folders/88/c8f0s39x66q5qc1fsgqmf62m0000gn/T/APKTOOL2620364219623722891.tmp, -0, arsc, -0, arsc, -I, /Users/raja/Library/apktool/framework/1.apk, -S, /Users/raja/cordovaTest/base/res, -M, /Users/raja/cordovaTest/base/AndroidManifest.xml]
	at brut.util.OS.exec(OS.java:95)
	at brut.androlib.res.AndrolibResources.aaptPackage(AndrolibResources.java:448)
	... 6 more
```

### Steps to Reproduce
1. `java -jar apktool_2.3.0.jar d base.apk `
2. `java -jar apktool_2.3.0.jar b base/ -o base.breaked.apk`

### APK
If this APK can be freely shared, please upload/attach a link to it.
https://drive.google.com/file/d/1kp8Xnq1cFuxXcMQMLkv5e6nsq1f2DgsI/view?usp=sharing

### Questions to ask before submission
1. Have you tried `apktool d`, `apktool b` without changing anything? yes
2. If you are trying to install a modified apk, did you resign it? n/a
3. Are you using the latest apktool version? yes

**Comment by Ahmed7fathi** [comment]
he's not gonna help you  with that issue 
what i know that is 448 is a cursed  code 
i spend about 4 days googling with that nothing helps

**Comment by Zane96** [comment]
@Ahmed7fathi  what do you men? I have meet the same issue

**Comment by Ahmed7fathi** [comment]
@Zane96  
its a bug

**Comment by Zane96** [comment]
@Ahmed7fathi 
but apktool empty-framework-dir --force this command is work for me!

**Comment by Ahmed7fathi** [comment]
@Zane96 
it works in some cases 
but there is an apps it didn't

**Comment by iBotPeaches** [comment]
Could not duplicate. This might already be fixed in the upcoming 2.3.1 release.

```
➜  1687 apktool d base.apk 
I: Using Apktool 2.3.1-ef9be8-SNAPSHOT on base.apk
I: Loading resource table...
I: Decoding AndroidManifest.xml with resources...
I: Loading resource table from file: /home/ibotpeaches/.local/share/apktool/framework/1.apk
I: Regular manifest package...
I: Decoding file-resources...
I: Decoding values */* XMLs...
I: Baksmaling classes.dex...
I: Baksmaling classes2.dex...
I: Copying assets and libs...
I: Copying unknown files...
I: Copying original files...
➜  1687 apktool b base
I: Using Apktool 2.3.1-ef9be8-SNAPSHOT
I: Checking whether sources has changed...
I: Smaling smali folder into classes.dex...
I: Checking whether sources has changed...
I: Smaling smali_classes2 folder into classes2.dex...
I: Checking whether resources has changed...
I: Building resources...
I: Copying libs... (/lib)
I: Building apk file...
I: Copying unknown files/dir...
➜  1687 
```
